src/test/java/com/rackspacecloud/metrics/tenantroutingservice/RoutingControllerUnitTest.java

```diff
@@ -72,25 +73,25 @@ private TenantRoutes getTenantRoutes() {
     public void test_getTenantRoutingInformation_validInput_returnsTenantRoutes() throws Exception {
         TenantRoutes output = getTenantRoutes();
 
-        when(routingServiceImpl.getIngestionRoutingInformation(anyString(), anyString())).thenReturn(output);
+        when(routingServiceImpl.getIngestionRoutingInformation(anyString(), anyString(), anyBoolean())).thenReturn(output);
 
-        TenantRoutes out = controller.getTenantRoutingInformation(anyString(), anyString());
+        TenantRoutes out = controller.getTenantRoutingInformation(anyString(), anyString(), anyBoolean());
 
         Assert.assertEquals("http://test-path:8086", out.getRoutes().get("full").getPath());
     }
 
     @Test
     public void test_GlobalExceptionHandler_postMethod_newTenant_throwsRouteWriteException() throws Exception {
         doThrow(RouteWriteException.class).when(routingServiceImpl)
-                .getIngestionRoutingInformation(anyString(), anyString());
+                .getIngestionRoutingInformation(anyString(), anyString(), anyBoolean());
 
         IngestionRoutingInformationInput input = new IngestionRoutingInformationInput();
         input.setDatabaseName("test_database");
         input.setPath("http://test-path:8086");
 
         ObjectMapper mapper = new ObjectMapper();
 
-        this.mockMvc.perform(get("/dummy_tenantId/measurement")
+        this.mockMvc.perform(get("/dummy_tenantId/measurement?readOnly=true")
```

Comment:
We should add another test case with url `/dummy_tenantId/measurement` so that we test the default value as well.